packages/cardpay-cli/index.ts

```diff
@@ -2,41 +2,72 @@
 import yargs from 'yargs';
 import fetch from 'node-fetch';
 import bridge from './bridge.js';
+import awaitBridged from './await-bridged.js';
 import viewSafes from './view-safes.js';
 import createPrepaidCard from './create-prepaid-card.js';
 
 //@ts-ignore polyfilling fetch
 global.fetch = fetch;
 
-type Commands = 'bridge' | 'safesView' | 'prepaidCardCreate';
+type Commands = 'bridge' | 'awaitBridged' | 'safesView' | 'prepaidCardCreate';
 
 let command: Commands | undefined;
 interface Options {
   network: string;
   mnemonic: string;
   tokenAddress?: string;
   amount?: number;
+  fromBlock?: number;
   address?: string;
   safeAddress?: string;
+  receiver?: string;
+  recipient?: string;
   amounts?: number[];
 }
-const { network, mnemonic = process.env.MNEMONIC_PHRASE, tokenAddress, amount, address, safeAddress, amounts } = yargs(
-  process.argv.slice(2)
-)
+const {
+  network,
+  mnemonic = process.env.MNEMONIC_PHRASE,
+  tokenAddress,
+  amount,
+  address,
+  safeAddress,
+  fromBlock,
+  receiver,
+  recipient,
+  amounts,
+} = yargs(process.argv.slice(2))
   .scriptName('cardpay')
   .usage('Usage: $0 <command> [options]')
-  .command('bridge <tokenAddress> <amount>', 'Bridge tokens to the layer 2 network', (yargs) => {
+  .command('bridge <amount> [tokenAddress]', 'Bridge tokens to the layer 2 network', (yargs) => {
+    yargs.positional('amount', {
+      type: 'number',
+      description: 'Amount of tokens you would like bridged (*not* in units of wei)',
+    });
     yargs.positional('tokenAddress', {
       type: 'string',
       default: '0x4F96Fe3b7A6Cf9725f59d353F723c1bDb64CA6Aa', // Kovan DAI
       description: 'The token address (defaults to Kovan DAI)',
     });
-    yargs.positional('amount', {
-      type: 'number',
-      description: 'Amount of tokens you would like bridged (*not* in units of wei)',
+    yargs.option({
```

Comment:
parhaps we make this positional to match the other args? I left the overarching args for all the commands as options.